### ISSUE TITLE
Team mates not available

### ISSUE THREAD

**Issue by Nedu** [Chingu-cohorts/voyage-wiki#16]
**_Issue Description & Expected Outcome:_** 
None of my team members have been added to our slack channel (bears-team-29).

I am not sure what my next steps should be now.

Thanks.
**_Symptoms:_**

**_Steps to Recreate:_** 

**_Resolution:_**

**Comment by jdmedlock** [comment]
The root cause is we have exceeded the maximum number of team member additions GitHub allows in a single day on 2/17/2018. We received one increase from GitHub Support late in the day and are currently waiting on another extension.

**Comment by jdmedlock** [comment]
Just reviewed with the team and it appears that the other two members of this team have not accepted the Slack invitation. Will look at placing Nedu in another team.

**Comment by Smithmayowa20** [comment]
Hello I am your team mate how do I accept the slack invite

**Comment by tropicalchancer** [comment]
Hi @Nedu!

@Smithmayowa20 and the other team-mate are in now. The team may need to make another ticket to give access to the Wizard.
